src/cephadm/cephadmlib/templates/agent.service.j2

```diff
@@ -0,0 +1,15 @@
+# generated by cephadm
+[Unit]
+Description=cephadm agent for cluster {{agent.fsid}}
+
+PartOf=ceph-{{agent.fsid}}.target
+Before=ceph-{{agent.fsid}}.target
+
+[Service]
+Type=forking
+ExecStart=/bin/bash {{agent.data_dir}}/unit.run
```

Comment:
Seems to be an issue with the agent running off of this. 
```
Oct 31 13:59:17 smithi092 systemd[1]: Starting cephadm agent for cluster e3a2336e-77ed-11ee-8db9-212e2dc638e7...
Oct 31 13:59:17 smithi092 bash[146780]: /bin/bash: /unit.run: No such file or directory
Oct 31 13:59:17 smithi092 systemd[1]: ceph-e3a2336e-77ed-11ee-8db9-212e2dc638e7@agent.smithi092.service: Control process exited, code=exited status=127
Oct 31 13:59:17 smithi092 systemd[1]: ceph-e3a2336e-77ed-11ee-8db9-212e2dc638e7@agent.smithi092.service: Failed with result 'exit-code'.
Oct 31 13:59:17 smithi092 systemd[1]: Failed to start cephadm agent for cluster e3a2336e-77ed-11ee-8db9-212e2dc638e7.
Oct 31 13:59:27 smithi092 systemd[1]: ceph-e3a2336e-77ed-11ee-8db9-212e2dc638e7@agent.smithi092.service: Service RestartSec=10s expired, scheduling restart.
Oct 31 13:59:27 smithi092 systemd[1]: ceph-e3a2336e-77ed-11ee-8db9-212e2dc638e7@agent.smithi092.service: Scheduled restart job, restart counter is at 4.
Oct 31 13:59:27 smithi092 systemd[1]: Stopped cephadm agent for cluster e3a2336e-77ed-11ee-8db9-212e2dc638e7.
Oct 31 13:59:27 smithi092 systemd[1]: Starting cephadm agent for cluster e3a2336e-77ed-11ee-8db9-212e2dc638e7...
Oct 31 13:59:27 smithi092 bash[146785]: /bin/bash: /unit.run: No such file or directory
Oct 31 13:59:27 smithi092 systemd[1]: ceph-e3a2336e-77ed-11ee-8db9-212e2dc638e7@agent.smithi092.service: Control process exited, code=exited status=127
Oct 31 13:59:27 smithi092 systemd[1]: ceph-e3a2336e-77ed-11ee-8db9-212e2dc638e7@agent.smithi092.service: Failed with result 'exit-code'.
Oct 31 13:59:27 smithi092 systemd[1]: Failed to start cephadm agent for cluster e3a2336e-77ed-11ee-8db9-212e2dc638e7.
```

```
[root@smithi092 ~]# cat /etc/systemd/system/ceph-e3a2336e-77ed-11ee-8db9-212e2dc638e7@agent.smithi092.service 
# generated by cephadm
[Unit]
Description=cephadm agent for cluster e3a2336e-77ed-11ee-8db9-212e2dc638e7

PartOf=ceph-e3a2336e-77ed-11ee-8db9-212e2dc638e7.target
Before=ceph-e3a2336e-77ed-11ee-8db9-212e2dc638e7.target

[Service]
Type=forking
ExecStart=/bin/bash /unit.run
Restart=on-failure
RestartSec=10s

[Install]
WantedBy=ceph-e3a2336e-77ed-11ee-8db9-212e2dc638e7.target
```
https://pulpito.ceph.com/adking-2023-10-31_04:11:54-orch:cephadm-wip-adk-testing-2023-10-30-1928-distro-default-smithi/7441777/